ci/gpu/build.sh

```diff
@@ -38,7 +38,7 @@ $CXX --version
 
 logger "Setup new environment..."
 conda install -c rapidsai/label/cuda$CUDA_REL -c rapidsai-nightly/label/cuda$CUDA_REL -c nvidia/label/cuda$CUDA_REL -c conda-forge \
```

Comment:
Channels are now set via a `.condarc` file in the build environment, so no need for that.
```suggestion
conda install \
```